backend/spec/features/admin/orders/listing_spec.rb

```diff
@@ -103,7 +113,8 @@
 
       # Regression test for #4004
       it "should be able to go from page to page for incomplete orders" do
-        10.times { Spree::Order.create email: "incomplete@example.com" }
+        Spree::Order.destroy_all
+        2.times { Spree::Order.create! email: "incomplete@example.com", completed_at: nil, state: 'cart' }
```

Comment:
Line is too long. [106/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.